integration/Jenkinsfile

```diff
@@ -180,8 +180,9 @@ pipeline {
 
 def void stopAndRemoveDockers() {
     sh '''
-        docker system prune --all --force --volumes
-
+        for container_id in $(docker ps -a -q);do docker stop $container_id && docker rm -f $container_id;done
+        for image_id in $(docker images -q);do docker rmi -f $image_id;done
+        for volume_id in $(docker volume ls -q); do docker volume rm -f $volume_id;done
```

Comment:
isn't this equal to docker volume prune?

Reply:
addressed